tests/lib/rules/lines-around-comment.js

```diff
@@ -317,8 +317,195 @@ ruleTester.run("lines-around-comment", rule, {
                 allowBlockEnd: true
             }],
             ecmaFeatures: {classes: true}
-        }
+        },
+
+        // check for object start comments
```

Comment:
Can you enlighten me as to what is complicated and what is not? Is that considered not complicated: https://github.com/eslint/eslint/blob/master/tests/lib/rules/lines-around-comment.js#L81 ?

If that's considered complicated however, do you want me to put the other complicated tests on multiple lines as well? Thanks!